src/box/lua/config/instance_config.lua

```diff
@@ -1404,13 +1404,24 @@ return schema.new('instance_config', schema.record({
         }, {
             default = 'off',
         }),
-        -- XXX: needs more validation
         peers = schema.array({
```

Comment:
We're out of 3.0 feature window now, so we shouldn't change `replication.peers` format/logic. I propose to add `replication.upstream` (or `replication.upstreams`?) that accepts a list of instance names.